targets/nrf5x/bluetooth.c

```diff
@@ -3100,6 +3100,9 @@ JsVar *jsble_get_security_status(uint16_t conn_handle) {
     jsvObjectSetChildAndUnLock(result, "encrypted", jsvNewFromBool(status.encrypted));
     jsvObjectSetChildAndUnLock(result, "mitm_protected", jsvNewFromBool(status.mitm_protected));
     jsvObjectSetChildAndUnLock(result, "bonded", jsvNewFromBool(status.bonded));
+#if NRF_MODULE_ENABLED(BLE_ADVERTISING)
```

Comment:
I suspect this isn't a macro issue, but that I need to ditch the `#if` and rebuild the local `nrf5x_12` component, then update the headers that this target includes (assuming they're done as part of the build process). Will look into this further

Reply:
Thanks, but I wouldn't worry about ifdef-ing this as all BLE devices will be able to advertise

Reply:
Cheers - not an issue with the new approach now too